index.html

```diff
@@ -7,46 +7,120 @@
           <label for="option1"></label>
-          <input id="option1" type="text" name="option1" />
-          <label><input type="radio" name="correct" value="0" />Correct</label
+          <input
+            id="option1"
+            type="text"
+            name="option1"
+            required
+            placeholder="option1"
+          />
+          <label
+            ><input
+              type="radio"
+              name="correct"
+              value="0"
+              required
+            />Correct</label
           ><br />
         </div>
         <div class="answer-option">
           <label for="option2"></label>
-          <input id="option2" type="text" name="option2" />
-          <label><input type="radio" name="correct" value="1" />Correct</label
+          <input
+            id="option2"
+            type="text"
+            name="option2"
+            required
+            placeholder="option2"
+          />
+          <label
+            ><input
+              type="radio"
+              name="correct"
+              value="1"
+              required
+            />Correct</label
           ><br />
         </div>
         <div class="answer-option">
           <label for="option3"></label>
-          <input id="option3" type="text" name="option3" />
-          <label><input type="radio" name="correct" value="2" />Correct</label
+          <input
+            id="option3"
+            type="text"
+            name="option3"
+            required
+            placeholder="option3"
+          />
+          <label
+            ><input
+              type="radio"
+              name="correct"
+              value="2"
+              required
+            />Correct</label
           ><br />
         </div>
         <div class="answer-option">
           <label for="option4"></label>
-          <input id="option4" type="text" name="option4" />
-          <label><input type="radio" name="correct" value="3" />Correct</label
+          <input
+            id="option4"
+            type="text"
+            name="option4"
+            required
+            placeholder="option4"
+          />
+          <label
+            ><input
+              type="radio"
+              name="correct"
+              value="3"
+              required
+            />Correct</label
           ><br />
         </div>
       </div>
       <div>
-        <label for="explanation">Explanation</label>
-        <input id="explanation" type="text" name="explanation" />
+        <label for="explanation"></label>
+        <input
+          id="explanation"
+          type="text"
+          name="explanation"
+          required
+          placeholder="Provide the explanation for the correct answer"
+        />
       </div>
       <div id="buttons">
         <button type="button" id="shuffle">Shuffle</button>
         <button type="submit" id="submit">Submit</button>
       </div>
     </form>
+    <section id="quizquestions"></section>
```

Comment:
In HTML/CSS then `kebab-case` is the most common

Reply:
Thanks for letting me know. I have changed it.